app/containers/markdown/index.js

```diff
@@ -317,7 +324,7 @@ export default class Markdown extends PureComponent {
 
 		// Ex: '[ ](https://open.rocket.chat/group/test?msg=abcdef)  Test'
 		// Return: 'Test'
-		m = m.replace(/^\[([\s]]*)\]\(([^)]*)\)\s/, '').trim();
+		m = m.replace(/(\s)\[([\s]]*)\]\(([^)]*)\)\s/, '').trim();
```

Comment:
What are you doing here?

Reply:
I saw that \s covers spaces and added it to regex and apparently worked for this case, but seeing better now, it is failing in the example above. Do you have any suggestion on how to solve this problem?